templates/upstart/default/init.conf

```diff
@@ -44,7 +44,6 @@ limit stack {{{limit_stack_size}}} {{{limit_stack_size}}}
 {{/limit_stack_size}}
 setuid {{{user}}}
 setgid {{{group}}}
-console log # log stdout/stderr to /var/log/upstart/
```

Comment:
I thought about this and think it would be useful to support either way, though don't know how to flag it.